src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+enum FigureType {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
-
+export interface Figure {
+  color: Color;
+  shape: FigureType;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = FigureType.Triangle;
```

Comment:
```suggestion
  readonly shape = FigureType.Triangle;
```
Figures have constant shape